### ISSUE TITLE
seems issues with softmax_cross_entropy_with_logits

### ISSUE THREAD

**Issue by argman** [tensorflow/tensorflow#1513]
matrix1 = tf.constant([[ -2.88181912e+11,-4.40300175e+11], [ -2.29688918e+11, -3.53027424e+11] ,[ -2.14650274e+11, -3.81851140e+11]])
matrix2 = tf.constant([[0, 1.0], [0, 1], [0, 1]])
r = tf.nn.softmax_cross_entropy_with_logits(matrix1, matrix2)

with tf.Session() as sess:
    result = sess.run(r)
    print result

when i run this, i get:

[  1.52118264e+11   1.23338506e+11   1.67200866e+11]

is this wrong ?

**Comment by mrry** [comment]
This seems like the intended behavior. You have a completely misclassified set of logits, and the cross entropy loss is very large.

What were you expecting to see instead?

**Comment by argman** [comment]
@mrry, I find this because I am running an AlexNet, and after a few iterations, i get Floating Point Exception, I guess this maybe the number is too large, can you give some advice to fix this?

**Comment by argman** [comment]
its the problem of my network parameters, sorry for this.
